File: B/Python/DataTim.py
# Kelas DataTim
class DataTim():
    __tim = ""
    __negara = ""
    __tahun = 0
    __pemain = ""

    # Constructor
    def __init__(self, tim = "", negara = "", tahun = 0, pemain  = ""):
        self.__tim = tim
        self.__negara = negara
        self.__tahun = tahun
        self.__pemain = pemain

    # Setter
    def setTim(self, tim):
        self.__tim = tim

    def setNegara(self, negara):
        self.__negara = negara

    def setTahun(self, tahun):
        self.__tahun = tahun

    def setPemain(self, pemain):
        self.__pemain = pemain

    # Getter
    def getTim(self):
        return self.__tim
    
    def getNegara(self):
        return self.__negara
    
    def getTahun(self):
        return self.__tahun

    def getPemain(self):
        return self.__pemain

    # Print Data
    def printData(self):
        print("Tim              : " + str(self.__tim))
        print("Negara           : " + str(self.__negara))
        print("Tahun Berdiri    : " + str(self.__tahun))
        print("Pemain           : " + str(self.__pemain))

File: B/PHP/DataTim.php
<?php

// Kelas DataTim
class DataTim
{
    private $tim = "";
    private $negara = "";
    private $tahun = 0;
    private $pemain = "";

    // Constructor
    public function __construct($tim = "", $negara = "", $tahun = 0, $pemain = "")
    {
        $this->tim = $tim;
        $this->negara = $negara;
        $this->tahun = $tahun;
        $this->pemain = $pemain;
    }

    // Setter
    public function setTim($tim)
    {
        $this->tim = $tim;
    }

    public function setNegara($negara)
    {
        $this->negara = $negara;
    }

    public function setTahun($tahun)
    {
        $this->tahun = $tahun;
    }

    public function setPemain($pemain)
    {
        $this->pemain = $pemain;
    }

    // Getter
    public function getTim()
    {
        return $this->tim;
    }

    public function getNegara()
    {
        return $this->negara;
    }

    public function getTahun()
    {
        return $this->tahun;
    }

    public function getPemain()
    {
        return $this->pemain;
    }

    // Print Data
    public function printData()
    {
        echo "Tim           : ". $this->getTim(). "<br/>";
        echo "Negara        : ". $this->getNegara(). "<br/>";
        echo "Tahun Berdiri : ". $this->getTahun(). "<br/>";
        echo "Pemain        : ". $this->getPemain(). "<br/>";
    }
}

?>